Release/src/http/client/http_client_named_pipe.cpp

```diff
@@ -261,17 +260,28 @@ class named_pipe_client : public _http_client_communicator
                 0,
                 NULL);
 
+			if (m_namedPipe != INVALID_HANDLE_VALUE)
+			{
+				break;
+			}
             auto errorCode = GetLastError();
-            if (m_namedPipe != INVALID_HANDLE_VALUE || errorCode != ERROR_PIPE_BUSY) 
-                break;
-
-            // Wait for pipe instance for 10 seconds
-            if(!WaitNamedPipe(pipe_name.c_str(), nTimeOut)) 
-            {
-                named_pipe_context->report_error(errorCode, build_error_msg(errorCode, "Could not open pipe: 10 second wait timed out."));
-                break;
-            }
-            retry_count--;
+			if (errorCode != ERROR_PIPE_BUSY)
+			{
+				named_pipe_context->report_error(errorCode, build_error_msg(errorCode, "Error opening named pipe"));
+				break;
+			}
+			else
+			{
+				// Wait for pipe instance to be avialabe, upto 10 sec(pipe server specified default)
+				if (!WaitNamedPipe(pipe_name.c_str(), NMPWAIT_USE_DEFAULT_WAIT))
+				{
+					if (retry_count <= 0)
+                    {
```

Comment:
can you fix the spacing here so that it lines up correctly? Maybe there's tab here instead of spaces?